### ISSUE TITLE
Problem with local replays saving.

### ISSUE THREAD

**Issue by norraxx** [FAForever/downlords-faf-client#1677]
I've been playing with lilsidlil player, and he asked me about: do i know anything about that replays are not saving at all on local machine?

Well, i've asked for logs and what i've found:
```
2020-04-22 21:17:18.282  WARN 14948 --- [Thread-1114] com.faforever.client.game.GameService    : Can't find 11556313 in gameInfoBean map
2020-04-22 21:17:29.974  INFO 14948 --- [Thread-1115] c.f.client.fa.relay.ice.IceAdapterImpl   : ICE adapter terminated normally
2020-04-22 21:17:30.259  INFO 14948 --- [pool-3-thread-590] com.faforever.client.game.GameService    : Forged Alliance terminated with exit code 0
```
Where 11556313 is the ID of the game.

I supose problem is in: `/faf-downlords-client/src/main/java/com/faforever/client/game/GameService.java`
```

  /**
   * Actually starts the game, including relay and replay server. Call this method when everything else is prepared
   * (mod/map download, connectivity check etc.)
   */
  private void startGame(GameLaunchMessage gameLaunchMessage, Faction faction, RatingMode ratingMode) {
    if (isRunning()) {
      log.warn("Forged Alliance is already running, not starting game");
      return;
    }

    stopSearchLadder1v1();
    int uid = gameLaunchMessage.getUid();
    replayServer.start(uid, () -> getByUid(uid)) // <<< HERE!
        .thenCompose(port -> {
          localReplayPort = port;
          return iceAdapter.start();
        })
```

Unfortunately can't tell you more.

Log is here:
[downlords-faf-client.log](https://github.com/FAForever/downlords-faf-client/files/4518943/downlords-faf-client.log)

**Comment by betrok** [comment]
It's fixed in current RC afaik.

**Comment by andreasley** [comment]
Yes, this is fixed in 1.1.6.
